iotdb-core/ainode/ainode/core/manager/inference_manager.py

```diff
@@ -50,32 +50,71 @@ def infer(self, full_data, **kwargs):
         pass
 
 
-# [IoTDB] full data deserialized from iotdb is composed of [timestampList, valueList, length],
-# we only get valueList currently.
+# [IoTDB] Full data deserialized from IoTDB is composed of [timestampList, valueList, length],
```

Comment:
I think all changes in this file is irrelevant with the goal of this PR, so restore this file